app/assets/javascripts/notifications.js

```diff
@@ -30,28 +30,19 @@ var onReadyNotifications = function() {
     }
   });
 
-  var click_flag = 0;
-
   resetNotificationsButton();
```

Comment:
It seems to me like we could maybe remove this call here, and then we wouldn't need to have the `!important` on the `reset_notification` class. I don't really understand why this is here in the first place so there might be something I am missing with regards to Pusher. @julianguyen should we make sure the opacity of the notification button is 1 when this JS loads, or are we okay to not do anything since the default opacity is 1...?